tokio-trace-fmt/src/default.rs

```diff
@@ -65,16 +65,16 @@ impl<'a> Recorder<'a> {
     }
 }
 
-impl<'a> ::NewRecorder<'a> for NewRecorder {
-    type Recorder = Recorder<'a>;
+impl<'a> ::NewVisitor<'a> for NewRecorder {
```

Comment:
is the name being switched to visitor?

Reply:
Yeah, `Record` is now the name of the type passed to `Subscriber::record`.